<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite thread-count="2" parallel="tests" name="Failed suite [Suite]" guice-stage="DEVELOPMENT">
  <test thread-count="2" parallel="tests" name="Test-Runner-2(failed)">
    <classes>
      <class name="contactsTests.CreateContact_LeadSourcxe">
        <methods>
          <include name="amConfig"/>
          <include name="bcConfig"/>
          <include name="bmConfig"/>
          <include name="acConfig"/>
          <include name="bsConfig"/>
          <include name="asConfig"/>
          <include name="createContactwithLead"/>
        </methods>
      </class> <!-- contactsTests.CreateContact_LeadSourcxe -->
    </classes>
  </test> <!-- Test-Runner-2(failed) -->
  <test thread-count="2" parallel="tests" name="Test-Runner-1(failed)">
    <classes>
      <class name="contactsTests.CreateContact_Assert">
        <methods>
          <include name="amConfig"/>
          <include name="bcConfig"/>
          <include name="bmConfig"/>
          <include name="acConfig"/>
          <include name="bsConfig"/>
          <include name="asConfig"/>
          <include name="CreateContactWithMandatoryField"/>
        </methods>
      </class> <!-- contactsTests.CreateContact_Assert -->
    </classes>
  </test> <!-- Test-Runner-1(failed) -->
</suite> <!-- Failed suite [Suite] -->
